spec/controllers/organizations_controller_spec.rb

```diff
@@ -1,4 +1,99 @@
 require 'rails_helper'
 
 RSpec.describe OrganizationsController, type: :controller do
+  describe 'GET #new' do
+    let(:make_request) {
+      get :new
+    }
+
+    before { make_request }
+
+    it 'has ok status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders new template' do
+      expect(response).to render_template(:new)
+    end
+  end
+
+  describe 'POST #create' do
+    context 'non-existant organization' do
+    end
+
+    context 'with a new organization' do
+      let(:make_request) {
+        VCR.use_cassette('github_organization') do
+          post :create, organization: {name: 'openlexington'}
+        end
+      }
+
+      it 'it assigns organization with a name' do
+        make_request
+        expect(assigns(:organization).name).to eq('openlexington')
+      end
+
+      it 'creates a new organization record' do
+        expect{make_request}.to change(Organization, :count).by(1)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.